src/components/Steps/StepWithOptions.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Parser from 'html-react-parser';
+import Parser, { domToReact } from 'html-react-parser';
```

Comment:
Is there a reason we need to import `domToReact` here? Doesn't seem to be something we're using

Reply:
Ah no I was trying previously but forgot to remove